### ISSUE TITLE
Error message can be more specific instead of giving a paragraph

### ISSUE THREAD

**Issue by TimothyLawSongEn** [TimothyLawSongEn/pe#4]
![Screenshot 2023-04-14 at 2.21.40 PM.png](https://raw.githubusercontent.com/TimothyLawSongEn/pe/main/files/424dd826-f4ac-4814-b876-d6275ac09d86.png)Screenshot 2023-04-14 at 2.21.40 PM

**Comment by nus-se-script** [comment]
# Team's Response
We agree that the paragraph is slightly too wordy and this makes it inconvenient to read. The number of words could indeed have been cut down to make it more concise. However, we disagree with your assessment of it being a functionality bug as it is actually a relatively minor cosmetic issue and not a bug. Thank you for raising this issue anyway!
# Items for the Tester to Verify
## :question: Issue type

Team chose [`type.FeatureFlaw`]
Originally [`type.FunctionalityBug`]

- [ ] I disagree

**Reason for disagreement:** [replace this with your explanation]

<catcher-end-of-segment><hr>
